From 1908ddeb28cd16e1d439eef885e2cc34dd89a7b6 Mon Sep 17 00:00:00 2001
From: Rogerio Nunes <rogerio.nunes@nxp.com>
Date: Wed, 27 Apr 2016 14:04:22 -0400
Subject: Makefile cleanup tagetting a Yocto Project recipe

Upstream status: pending 

Signed-off-by: Rogerio Nunes <rogerio.nunes@nxp.com>

diff --git a/Makefile b/Makefile
index 79a4ef9..a6d63d7 100644
--- a/Makefile
+++ b/Makefile
@@ -1,23 +1,16 @@
-KERNELRELEASE ?= $(shell uname -r)
-BUILD_KERNEL_PATH ?= /lib/modules/$(KERNELRELEASE)/build
-INSTALL_MOD_PATH ?= $(INSTALL_PATH)/lib/modules/$(KERNELRELEASE)
-INSTALL_PN5XX_I2C_PATH ?= $(INSTALL_MOD_PATH)/kernel/drivers/nfc
-
 obj-m := pn5xx_i2c.o
-nxp-pn5xx-objs := pn5xx_i2c.o
 
-all: pn5xx_i2c.ko
+DESTDIR ?= $(INSTALL_MOD_PATH)
 
-pn5xx_i2c.ko: pn5xx_i2c.c pn5xx_i2c.h
-	@echo "Compiling kernel module"
-	$(MAKE) -C $(BUILD_KERNEL_PATH) M=$(CURDIR) modules
+modules all:
+	$(MAKE) -C $(KERNEL_SRC) M=$(CURDIR) modules
 
-install: all
-	mkdir -p $(INSTALL_PN5XX_I2C_PATH)
-	install -m 644 pn5xx_i2c.ko $(INSTALL_PN5XX_I2C_PATH)
+modules_install install: all
+	$(MAKE) -C $(KERNEL_SRC) M=$(CURDIR) modules_install \
+		INSTALL_MOD_PATH=$(DESTDIR) INSTALL_MOD_DIR="kernel/drivers/misc"
 
 clean:
-	$(MAKE) -C $(BUILD_KERNEL_PATH) M=$(CURDIR) clean
+	$(MAKE) -C $(KERNEL_SRC) M=$(CURDIR) clean
 
 .PHONY: all install clean
 .EXPORT_ALL_VARIABLES:
-- 
1.9.1

